guides/common/modules/proc_configuring-podman-to-trust-the-ca.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Configuring_podman_to_trust_the_CA_{context}"]
+= Configuring Podman and Docker to trust the Certificate Authority
+
+Podman uses two paths to locate the CA file, namely, `/etc/containers/certs.d/` and `/etc/docker/certs.d/`.
+
+* Copy the root CA file to one of these locations, with the exact path determined by the server hostname, and naming the file `ca.crt`:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ sudo cp rootCA.pem /etc/containers/certs.d/quay-server.example.com/ca.crt
+----
+
+* Alternatively, if you are using Docker, you can copy the root CA file to the equivalent Docker directory:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ sudo cp rootCA.pem /etc/docker/certs.d/quay-server.example.com/ca.crt
+----
+
+You should no longer need to use the `--tls-verify=false` option when logging in to the registry:
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ sudo podman login quay-server.example.com
```

Comment:
Here I'm wondering: I guess that podman as non-root will also work and use the certificates from `/etc`. So would `$ podman` also work (as opposed to `# podman` / `$ sudo podman`)?

Reply:
I'll check with @jlsherrill

Reply:
yes, no need for sudo here